src/main/java/core/basesyntax/BuffWriter.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class BuffWriter {
+    private String fileToRead;
+    private String fileToWrite;
```

Comment:
it would be more convenient if we could pass this fields as arguments to our methods